bundles/config/org.eclipse.smarthome.config.discovery.usbserial.linuxsysfs/src/main/java/org/eclipse/smarthome/config/discovery/usbserial/linuxsysfs/internal/SysfsUsbSerialScanner.java

```diff
@@ -142,8 +147,12 @@ protected void modified(Map<String, Object> config) {
         return result;
     }
 
-    private boolean isAccessible(Path devicePath) {
-        return exists(devicePath) && isWritable(devicePath) && isReadable(devicePath);
+    private boolean isReadWriteAccessible(Path path) {
+        return isReadAccessible(path) && isWritable(path);
+    }
+
+    private boolean isReadAccessible(Path path) {
+        return exists(path) && isReadable(path);
```

Comment:
isReadable checks already if the path exists...
```
This method checks that a file exists and that this Java virtual machine has appropriate privileges that would allow it open the file for reading.
```

Reply:
Thanks for the hint, I will improve the code accordingly.